### ISSUE TITLE
Can't enter longitude greater than 90°

### ISSUE THREAD

**Issue by bpmackow** [hvdwolf/jExifToolGUI#304]
**Version (please complete the following information):**
 - OS: macOS
 - Version: 13.2.1 Ventura
 - Package: macos bundle with jre (version 2.0.2)

**Describe the bug**
I can't add a longitude degree value greater than 90 (that's literally half the planet) without converting to decimal degrees

**To Reproduce**

1. Open some image
2. Edit data > GPS > Deg-Min-Sec Format
3. Enter a value greater than or equal to 90 (say, 111) in the longitude deg box
4. Click "save to selected image(s)"
5. Error message saying "degrees must be between 0 and <90"
6. Image does not save

**Comment by hvdwolf** [comment]
There are even multiple bugs in that screen.
I will have a look at that.

**Comment by hvdwolf** [comment]
I corrected the error for empty fields in the source code, but I can't find the error with the longitude.
The longitude value is between 0 and 180 and checked for 0 to 180.

This might take a little longer to find.

**Comment by hvdwolf** [comment]
An oldie but a goldie from Dr. Who and the Daleks. :)

![daleks-universe](https://user-images.githubusercontent.com/4520361/229295651-41ad733e-aa3d-4507-8372-ef640e3d8d47.jpg)

**Comment by lordofscripts** [comment]
Has this repository been abandoned?

**Comment by hvdwolf** [comment]
Yes, The Master readme also mentions that: https://github.com/hvdwolf/jExifToolGUI
Sorry, but it is what it is.
